Automation hub, team and org scoped automations (#2959)
wandb/docs: snippets/_includes/automations/where-to-find-automations.mdx

> - **In the Automations hub**: Select **Automations** in the left navigation to see every automation you can access across all scopes and projects. You choose the scope (Organization, Team, Registry, Registry collection, or project) when you create the automation. See [Automations hub](/models/automations#automations-hub).

Author: ```suggestion
- **In the Automations hub**: Select **Automations** in the left navigation to see every automation you can access across all scopes: organization, team, project, registry, or collection. You choose the scope when you create an automation. See [Automations hub](/models/automations#automations-hub).
```
